BoxContentSDK/BoxContentSDK/Requests/BOXRecentItemsRequest.m

```diff
@@ -68,10 +68,22 @@ - (void)performRequestWithCompletion:(BOXRecentItemsBlock)completionBlock
             NSArray *recentsJSON = JSONDictionary[BOXAPICollectionKeyEntries];
             NSMutableArray<BOXRecentItem *> *mutableRecentItems = [NSMutableArray arrayWithCapacity:recentsJSON.count];
 
+            BOXSharedLinkHeadersHelper *sharedLinkHelper = [[BOXContentClient defaultClient] sharedLinksHeaderHelper];
```

Comment:
super minor - no need to access the `sharedLinkHelper` until we're in the block after `recentItem.sharedLinkURL != nil)`